src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -449,6 +453,28 @@ private void cancelPatronHoldWithClosedRequest(InnReachTransaction transaction)
     }
   }
 
+  private void clearPatronTransactionAndItemRecord(UUID itemId, InnReachTransaction transaction) {
+    var patronTransaction = (TransactionPatronHold) transaction.getHold();
+    patronTransaction.setPatronId(null);
```

Comment:
There is `InnReachTransactionUtils.clearPatronAndItemInfo()` to help with removing patron info. It can be reused here

Reply:
Done